ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource

```diff
@@ -368,3 +369,14 @@ Wait For Job With Status
     [Arguments]    ${job_name}    ${job_status}    ${timeout}
     Wait Until Element Is Visible    //td[@class='pf-v5-c-table__td' and contains(text(), '${job_name}')]/following-sibling::td[@data-label='Status']/span/span/span[@class='pf-v5-c-label__text'and text()='${job_status}']    timeout=${timeout}
 
+Check Warning Threshold Chart
+    [Documentation]    verifies warning threshold of cpu and memory resources
+    Mouse Over    xpath=(//*[name()='svg']//*[local-name()='g']//*[local-name()='path'])[4]
+    ${hover_cpu_warning} =  Get Requested Resources Tooltip Text
+    Should Match   ${hover_cpu_warning}   Requested resources have surpassed 150%
+    Mouse Over    xpath=//*[@id="requested-resources-chart-CPU-ChartLegend-ChartLabel-0"]
```

Comment:
Why this mouse over is needed ?

Reply:
This mouse hover is occurring outside the chart because, after the first hover, there is a delay of a few seconds before it disappears, which interferes with the subsequent hover actions.